File: content/code/fahrenheit_to_celsius_docstring.jl
"""
    fahrenheit_to_celsius(temp_f::Float)

Converts temperature in Fahrenheit to Celsius.

# Arguments
- `temp_f::Float`: Temperature in Fahrenheit.

# Returns
- `temp_c::Float`: Temperature in Celsius.
"""
function fahrenheit_to_celsius(temp_f)
    temp_c = (temp_f - 32.0) * (5.0/9.0)
    return temp_c
end

File: content/code/fahrenheit_to_celsius_docstring.rs
/// Convert Fahrenheit to Celsius
/// # Arguments
/// * `temp_f` - Temperature in Fahrenheit
///
/// # Returns
/// * `temp_c` - Temperature in Celsius
fn fahrenheit_to_celsius(temp_f: f64) -> f64 {
    let temp_c = (temp_f - 32.0) * 5.0 / 9.0
    temp_c
}
